docs/api-reference/autocomplete.mdx

```diff
@@ -0,0 +1,190 @@
+            FROM tokenized
+        )]
+    )
+);
+```
+
+<Accordion title="Expected Response">
+```csv
+     description     | rating | category
+---------------------+--------+----------
+ Sleek running shoes |      5 | Footwear
+ White jogging shoes |      3 | Footwear
+ Generic shoes       |      4 | Footwear
+(3 rows)
+```
+</Accordion>
+
+## Multiple Fuzzy Fields
+
+Suppose we want to compare a query against both `description` and `category`. The `boolean` query can be used to query across multiple fields.
+
+```sql
+-- Pre-tokenize "ruining shoez"
+WITH tokenized AS (
+    SELECT token FROM paradedb.tokenize(
+        paradedb.tokenizer('default'),
+        'ruining shoez'
+    )
+)
+-- Match a document if any of the tokens match
+SELECT description, rating, category FROM search_idx.search(
+    query => paradedb.boolean(
+        should => ARRAY[
+            (
+                SELECT array_agg(paradedb.fuzzy_term(field => 'description', value => token))
+                FROM tokenized
+            ),
+            (
+                SELECT array_agg(paradedb.fuzzy_term(field => 'category', value => token))
+                FROM tokenized
+            )
+        ]
+    )
+);
+```
+
+<Accordion title="Expected Response">
+```csv
+     description     | rating | category
+---------------------+--------+----------
+ Sleek running shoes |      5 | Footwear
+ White jogging shoes |      3 | Footwear
+ Generic shoes       |      4 | Footwear
+(3 rows)
+```
+</Accordion>
+
+## Ngram Term
+
+Suppose we want to suggest results when the user has only typed part of a word, like `sho`. In this scenario,
+the [ngrams tokenizer](/api-reference/indexing/create_index#ngrams) can be used to convert documents into ngram tokens.
+
+For the purpose of this example, let's assume that we have an index called `ngrams_idx`:
+
+```sql
+CALL paradedb.create_bm25(
+    index_name => 'ngrams_idx',
+    schema_name => 'public',
+    table_name => 'mock_items',
+    key_field => 'id',
+    text_fields => paradedb.field(
+        'description',
+        tokenizer => paradedb.tokenizer('ngram', min_gram => 3, max_gram => 3, prefix_only => false)
+    )
+);
+```
+
+With `description` tokenized into n-grams, we can search for partial words.
+
+```sql
+SELECT description, rating, category
+FROM ngrams_idx.search('description:sho');
+```
+
+<Accordion title="Expected Response">
+```csv
+     description     | rating | category
+---------------------+--------+----------
+ Sleek running shoes |      5 | Footwear
+ White jogging shoes |      3 | Footwear
+ Generic shoes       |      4 | Footwear
+(3 rows)
+```
+</Accordion>
+
+## Ngram Term Set
+
+When querying against an ngrams field, all ngrams of the query must match in order for the document to be considered a match.
+This means that a query like `hsoes` would not match `shoes`, because the `hso` token does not match any of the tokens of `shoes`.
```

Comment:
It might be useful to link this in this subsection too: [ngrams tokenizer](/api-reference/indexing/create_index#ngrams)